src/features/flows/nRF9151/index.ts

```diff
@@ -0,0 +1,319 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-4-Clause
+ */
+
+import Apps from '../../../common/steps/Apps';
+import Develop from '../../../common/steps/develop';
+import Evaluate from '../../../common/steps/Evaluate';
+import Info from '../../../common/steps/Info';
+import Learn from '../../../common/steps/Learn';
+import Program from '../../../common/steps/program';
+import { Choice } from '../../../common/steps/program/programSlice';
+import Rename from '../../../common/steps/Rename';
+import Verify from '../../../common/steps/verify';
+import SimCard from './SimCard';
+
+const infoConfig = {
+    title: 'Massive IoT Powerhouse',
+    markdownContent:
+        '![nRF9151 DK](9151DK.png)  \n&nbsp;  \nThe nRF9151 Development Kit is perfect for evaluating the nRF9151 SiP and developing cellular IoT or DECT NR+ applications.  \n&nbsp;  \nIt includes a SEGGER J-Link OB debugger and all the necessary external circuirty like (e)SIM interface, antennas, access to all IO pins, programmable buttons, and module interfaces.  \n&nbsp;  \n[Hardware documentation](https://docs.nordicsemi.com/bundle/ncs-latest/page/zephyr/boards/arm/nrf9151dk_nrf9151/doc/index.html)  \n&nbsp;  \n![nRF9151 SiP Cores](9151Cores.png)  \nYou have two options for leveraging the nRF9151 SiP:  \n&nbsp;  \n**Option 1** *Recommended*  \nHarness the full potential of nRF9151 by using the entire application core for your code, eliminating the need for an external MCU, and fully capitalizing on the advantages offered by nRF9151.  \n&nbsp;  \n**Option 2**  \nIntegrate nRF9151 as a [cellular](https://www.nordicsemi.com/Products/Low-power-cellular-IoT/What-is-cellular-IoT#infotabs) or [NR+](https://www.nordicsemi.com/Products/DECT-NR) modem alongside your existing design running a serial AT command set on the application core.',
```

Comment:
```suggestion
        '![nRF9151 DK](9151DK.png)  \n&nbsp;  \nThe nRF9151 Development Kit is perfect for evaluating the nRF9151 SiP and developing cellular IoT or DECT NR+ applications.  \n&nbsp;  \nIt includes SEGGER's J-Link OB debug probe and all the necessary external circuirty, like (e)SIM interface, antennas, access to all IO pins, programmable buttons, and module interfaces.  \n&nbsp;  \n[Hardware documentation](https://docs.nordicsemi.com/bundle/ncs-latest/page/zephyr/boards/arm/nrf9151dk_nrf9151/doc/index.html)  \n&nbsp;  \n![nRF9151 SiP Cores](9151Cores.png)  \nYou have two options for leveraging the nRF9151 SiP:  \n&nbsp;  \n**Option 1** *Recommended*  \nHarness the full potential of nRF9151 by using the entire application core for your code, eliminating the need for an external MCU, and fully capitalizing on the advantages offered by nRF9151.  \n&nbsp;  \n**Option 2**  \nIntegrate nRF9151 as a [cellular](https://www.nordicsemi.com/Products/Low-power-cellular-IoT/What-is-cellular-IoT#infotabs) or [NR+](https://www.nordicsemi.com/Products/DECT-NR) modem alongside your existing design running a serial AT command set on the application core.',
```
Re https://www.segger.com/products/debug-probes/j-link/models/j-link-ob/

Reply:
Forgot to correctly point this pr to another branch so these changes aren't really supposed to be visible here.
I'll adjust it in the other branch though.
Same goes for your other comments below